.golangci.yml

```diff
@@ -1,3 +1,12 @@
+version: 2
+linters:
+  exclusions:
+    rules:
+    - path: _test\.go
+      linters:
+      - gosec
+      - goconst
+# v1 config, remove when golangci-lint v2 is active
```

Comment:
mm.. so when can we remove this? aren't we just making v2 active with this PR?

Reply:
Nope, this is preparing for the rollout of v2. Team Honeybadger will do the rollout at a later date.